### PR TITLE
Improve Identity Verification Notebook

### LINKED ISSUE #31: New library needed
For "identity-verification/module_1" the first command in the SageMaker Jumpstart needs a new library. The error does provide guidance on the new library to use.

now: !pip install boto3 matplotlib opencv-python openpyxl pandas sklearn scikit-image
required: !pip install boto3 matplotlib opencv-python openpyxl pandas scikit-learn scikit-image

The error: The 'sklearn' PyPI package is deprecated, use 'scikit-learn'

### PR DESCRIPTION
**Pull request by ConnorKirk**
* Update package name  from sklearn to scikit-learn
* Improve the section for creating a bucket for the workshop

Resolves #31, #30

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb [hunk 1]
```diff
@@ -54,28 +54,34 @@
...
+      "AWS Region: eu-west-2\n",
+      "AWS Bucket: ckp-packaging-us-east-1\n",
+      "Stored 'bucket_name' (str)\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/ckp/code/work/github.com/aws-samples/amazon-rekognition-code-samples/.venv/lib/python3.11/site-packages/IPython/extensions/storemagic.py:229: UserWarning: using autorestore/bucket_name requires you to install the `pickleshare` library.\n",
```
Comment (svozza) [inline identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb]: Are these local directory paths meant to be here?

Comment (matteofigus) [reply identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb]: +1 - I guess you want the output to be wiped from source.

**Review by matteofigus — CHANGES_REQUESTED** [review]

File: identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb (hunk 1 above)
Comment (ConnorKirk) [reply identity-verification/module_1/1.0 SETUP - Gather images and setup S3 buckets for IDV modules.ipynb]: Good spot. Thank you! I have removed the outputs

**Review by svozza — APPROVED** [review]

**Review by matteofigus — APPROVED** [review]

**Comment by matteofigus** [comment]
Thanks @ConnorKirk good work.

**Pull request merged**